src/index.js

```diff
@@ -198,21 +198,33 @@ const connectCallReceiver = (info, methods, destructionPromise) => {
 
             log(`${localName}: Sending ${methodName}() reply`);
 
+            if (returnValue instanceof Error) {
+              setTimeout(() => {
+                throw returnValue;
+              });
+              returnValue = returnValue.toString();
+            }
+
             remote.postMessage({
               penpal: REPLY,
               id,
               resolution,
               returnValue,
             }, remoteOrigin);
-          }
+          };
         };
 
         new Penpal.Promise((resolve, reject) => {
           try {
             // The consumer's function may throw an error, return a raw return value, return
             // a promise that resolves, or return a promise that gets rejected.
             // In the case that it throws an error, we'll send the error stack as a reply
-            resolve(methods[methodName](...args));
+            const result = methods[methodName](...args);
+            if (result instanceof Error) {
```

Comment:
Could you describe why someone would want to return an error from their method rather than just throw it?